lecture/views.py

```
@@ -31,19 +67,46 @@ def lecture_home(request: HttpRequest) -> HttpResponse:
     form = LectureForm(request.POST or None)
     if request.method == "POST" and form.is_valid():
-        form.save()
+        module_instance=form.save(commit=False)
+        module_instance.course=course_instance
+        module_instance.save()
         return redirect(f'{reverse("lecture_home")}?slug={slug}')
-    return render(request, "lecture/create_form.html", {"form": form, 'form_type': 'lecture'})
+    return render(request, "lecture/create_form.html", {"form": form, 'form_type': 'module'})
+
+
+def lecture_video_content(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `lecture_video_content`

(ANN201)